samples/net/sockets/echo_async/README.rst

```diff
@@ -78,6 +79,7 @@ To test:
 .. code-block:: console
 
     $ telnet 127.0.0.1 4242
+    $ telnet ::1 4242
```

Comment:
Yes, they're intended to be run sequentially (with extra interaction for each), or in different terminal windows. Added comments (for similar commands earlier in the doc too).